_episodes/05-snakemake.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Snakemake"
+teaching: 30
+exercises: 20
+questions:
+- "What are computational workflows?"
+- "How do you program using a build system?"
+- "How do I mix Python code into a workflow?"
+objectives:
+- "Understand the operating logic (or semantics) of Snakemake."
+- "Create a parallel workflow from an existing script."
+keypoints:
+- "Snakemake is ideal for specifying workflows with large granularity."
+- "Dependencies are automatically resolved; Snakemake is 'demand driven' programming."
+- "You may freely intermix shell commands with Python code."
+- "Results are persistent: you can resume computations when needed."
+---
+
+> ## Metalesson!
+> This is a metalesson! The instructor should fill in the details.
+>
+> The material in this demo is currently geared towards an astrophysics application. It is not essential that you understand the specifics of this application. That being said: it is much encouraged to use an example that the instructor is familiar with. The choice of a "real world" example is deliberate: we want participants to feel that this is a real application with real benifits. If you use a different demo, **please consider contributing** it to our repository. Ideally, future instructors can then choose a prepared demo that best suits their taste or background.
+{: .callout}
+
+# Setup
+> ## Preparation
+> To follow this tutorial, participants need to have cloned the repository at [github.com/escience-academy/parallel-python-workshop](https://github.com/escience-academy/parallel-python-workshop/). This repository includes an `environment.yml` file for creating the proper Conda environment, as well as the Jupyter notebook and `Snakefile` used in this demo.
+> This environment includes the GraphViz and ImageMagick packages needed to do some of the visualizations in this tutorial.
```

Comment:
Could be nice to add a link to Graphviz (https://graphviz.org/) and ImageMagick (https://imagemagick.org/).

Reply:
Solved in commit https://github.com/carpentries-incubator/lesson-parallel-python/pull/45/commits/b2a0585c86faeaa825db49b6ba7f19e87cd78450